src/main/cms/cms_menu_imu.c

```diff
@@ -62,13 +62,13 @@
 //
 static uint8_t tmpPidProfileIndex;
 static uint8_t pidProfileIndex;
-static char pidProfileIndexString[] = " p";
+static char pidProfileIndexString[MAX_PROFILE_NAME_LENGTH + 5];
```

Comment:
+5 ?

Reply:
The profile string is formatted like `9 (AAAAAAAA)` so the +5 is to make room in the buffer for the other characters beyond the profile name max size.